src/cgroup_builder.rs

```diff
@@ -80,6 +80,7 @@ pub struct CgroupBuilder {
     name: String,
     /// Internal, unsupported field: use the associated builders instead.
     resources: Resources,
+    specified_controllers: Option<Vec<String>>,
```

Comment:
Cloud you add some comment for this new value in `pub struct` like why do we need a `specified controllers`

Reply:
added